### ISSUE TITLE
document is not defined error shown

### ISSUE THREAD

**Issue by OblivionSY** [kingscode/vuetify-phone-input#3]
This may well be me missing something on the installation.

I ran 

`npm install --save @kingscode/vuetify-phone-input`

I created a plugin.js file

```
import Vue from 'vue'
import VuetifyPhoneInput from '@kingscode/vuetify-phone-input'

Vue.use(VuetifyPhoneInput)
```

Added plugin file to nuxt.config

```
  plugins: [
    '@/plugins/base-axios.js',
    '@/plugins/common-shared.js',
    '@/plugins/vuetify-phone-input.js',
  ],
```

When running the application now:
![image](https://user-images.githubusercontent.com/3511993/99184198-260df480-2739-11eb-9097-70d5227ec588.png)

If I remove it from the plugins list, the app works fine again. I have not done anything more than install and create the plugin, but maybe there is something extra needed for nuxt? Any thoughts please?

**Comment by ricardovanlaarhoven** [comment]
I've never used nuxt...
For another issue i'm removing the installation part on this component so you can just import the component yourself.
Otherwise you can't overwrite sass variables because vuetify is loaded twice.

for now you could try not installing this component and just locally import it inside the component you need it

**Comment by ricardovanlaarhoven** [comment]
BTW by looking at your screenshot i realised that this isn't vuetify-phone-input code, but vue or vuetify code.
Perhaps the next version will fix things for you anyway

**Comment by ricardovanlaarhoven** [comment]
There is a new version, you could check this version

**Comment by Tobeyforce** [comment]
I tried the same method as OP and still could not get this to work on Nuxt. Any chance you could take some time to explore how this would be done? Would be greatly appreciated @ricardovanlaarhoven 
@OblivionSY Did you give up on this?

**Comment by ricardovanlaarhoven** [comment]
Could you create a reproduction?

Ps. the new version has a new installation way. Did you use that way? (see readme)

**Comment by OblivionSY** [comment]
Unfortunately I just reverted to a TextField with a regex validation. It is something I'd like to improve, but it was a nice to have, rather than must have feature so I moved on.

**Comment by Tobeyforce** [comment]
> Could you create a reproduction?
> 
> Ps. the new version has a new installation way. Did you use that way? (see readme)

Yes I went with the installation way but still nothing that worked for me.
Unfortunately I don't have time to reproduce it, just like @OblivionSY this is a nice to have but I used vee validate instead with a custom validator. However, since phone fields are common I can see how this would be popular if it worked out of the box with Nuxt.

**Comment by ricardovanlaarhoven** [comment]
I've never really worked with nuxt, so id'have to invest time in that as well, and unfortunately i don't have that much time at this moment.

If someone could create a reproduction i could take a look :)
